lms/djangoapps/affiliates/views.py

```diff
@@ -57,7 +57,7 @@ def create(request):
     affiliate = AffiliateEntity()
     post_data = request.POST.copy()
 
-    program_director_identifier = post_data.pop('member_identifier', None)[0]
+    program_director_identifier = post_data.pop('member_identifier', [None])[0]
```

Comment:
Prevent blowing up when member_identifier is not present. I'm open to better solutions.